### DISCUSSION TITLE
OpenTelemetry JSON over HTTP support

### DISCUSSION THREAD

**Discussion by rudranil-das** [jaegertracing/jaeger#3739] [Q&A]
Is there a plan or timeline for jaeger to support OpenTelemetry JSON over HTTP ?

Due to restrictions in the production network I am using, it is not possible to deploy a jaeger service end-point.
So the thought is - 
a. dump the traces in Otel json format in container logs
b. collect the traces from the logs through a native agent and archive
c. on need basis, outside production network
     1. deploy a jaeger end-point
     2. make HTTP/REST calls using Otel json traces from archive to jaeger end-point (to get the visualisation)

Otel json format would be relatively easy to deal with for 
- interpretation in logs and 
- offline communication with jaeger end-point straight from archived data.

**Comment by yurishkuro** [comment]
Currently we're using OTLP receiver code from OTEL Collector contrib. If it supports json then Jaeger will automatically support json.
